roles/oneagent/tests/conmponent/scripts/util/test_helpers.py

```diff
@@ -0,0 +1,97 @@
+import functools
```

Comment:
refactor the file location to not use `util` directory and name the file with a descriptive name

Reply:
let's move it to the backlog